packages/esm-service-queues-app/src/views/queue-tables-for-all-statuses.component.tsx

```diff
@@ -44,50 +46,52 @@ const QueueTablesForAllStatuses: React.FC<QueueTablesForAllStatusesProps> = ({ s
   }
 
   return (
-    <div className={styles.container}>
-      <div className={styles.headerContainer}>
-        <div className={isDesktop(layout) ? styles.desktopHeading : styles.tabletHeading}>
-          <h3>{selectedQueue.display}</h3>
-        </div>
-        <div>
-          <QueueTableMetrics selectedQueue={selectedQueue} />
+    <>
+      <PatientQueueHeader title={selectedQueue?.display} hideLocationDropdown />
+      <QueueTableMetrics selectedQueue={selectedQueue} />
+
+      {/* <MetricsHeader /> */}
+      <div className={styles.container}>
+        <div className={styles.headerContainer}>
+          <div className={styles.headerButtons}>
+            <ExtensionSlot
+              name="patient-search-button-slot"
+              state={{
+                buttonText: t('addPatientToQueue', 'Add patient to queue'),
+                overlayHeader: t('addPatientToQueue', 'Add patient to queue'),
+                buttonProps: {
+                  kind: 'secondary',
+                  renderIcon: (props) => <Add size={16} {...props} />,
+                  size: isDesktop(layout) ? 'sm' : 'lg',
+                },
+                selectPatientAction: (selectedPatientUuid) => {
+                  setShowOverlay(true);
+                  setViewState({ selectedPatientUuid });
+                },
+              }}
+            />
+            <div className={styles.filterSearch}>
+              <Search
+                labelText=""
+                placeholder={t('filterTable', 'Filter table')}
+                onChange={(e) => setSearchTerm(e.target.value)}
+                size={isDesktop(layout) ? 'sm' : 'lg'}
```

Comment:
The size here appears backwards.  Shouldn't this be 'lg' if desktop, and 'sm' otherwise?

Reply:
No @mseaton , the components are bigger in tablet view, given the precision of cursor vs finger.

Reply:
Oh, ok.  The search box seems too small to me.  If this is normal, that's ok I guess.  But it seems strangely small.